From wfp5p@virginia.edu  Tue Apr 28 15:58:11 2009
From: Bill Pemberton <wfp5p@virginia.edu>
Date: Sun, 26 Apr 2009 14:45:11 -0400
Subject: Staging: comedi: replace for loop with msleep()
To: greg@kroah.com
Cc: abbotti@mev.co.uk, fmhess@users.sourceforge.net, ds@schleef.org
Message-ID: <1240771512-31483-2-git-send-email-wfp5p@virginia.edu>


Replace 2 attempts to use a for loop as a sleep with a call to msleep().

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/addi-data/addi_amcc_S5920.c |    2 +-
 drivers/staging/comedi/drivers/addi-data/hwdrv_apci3200.c  |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/comedi/drivers/addi-data/addi_amcc_S5920.c
+++ b/drivers/staging/comedi/drivers/addi-data/addi_amcc_S5920.c
@@ -180,7 +180,7 @@ int i_AddiHeaderRW_ReadEeprom(int i_NbOf
 			}
 
 			/* Sleep */
-			for (i = 0; i < 10000; i++) ;
+			msleep(1);
 
 		}
 		w_ReadWord =
--- a/drivers/staging/comedi/drivers/addi-data/hwdrv_apci3200.c
+++ b/drivers/staging/comedi/drivers/addi-data/hwdrv_apci3200.c
@@ -224,7 +224,7 @@ int i_AddiHeaderRW_ReadEeprom(int i_NbOf
 
 
 			/* Sleep */
-			for (i = 0; i < 10000; i++) ;
+			msleep(1);
 
 		}
 		w_ReadWord =
